dracula-theme.el

```diff
@@ -411,12 +411,34 @@ read it before opening a new issue about your will.")
                ;; flyspell
                (flyspell-duplicate :underline (:style wave :color ,dracula-orange))
                (flyspell-incorrect :underline (:style wave :color ,dracula-red))
-               ;; font-latex
+               ;; auctex
```

Comment:
To keep things consistent, can you please restore the comment `;; font-latex` but suffixing it like this: `;; font-latex (auctex)`. Then move all `TeX-*` faces definition to the bottom of the file (take care of the alphabetical order) under a new comment/title `;; TeX (auctex)` ?

Thank you